interactive-bots/helloworld/KubsHD/notes.md

```diff
@@ -0,0 +1 @@
+The only problem with this project, was that the bot crashed for some unknown reason, tools/do-destroy-rebuild-database fixed it.
```

Comment:
Can you line wrap all text files to 79 characters?

Reply:
@KubsHD You are requested to break this line into two for better readability. :)

Reply:
Okay, I changed it.